src/pip/_internal/cli/autocompletion.py

```diff
@@ -100,6 +100,12 @@ def autocomplete() -> None:
             if option[1] and option[0][:2] == "--":
                 opt_label += "="
             print(opt_label)
+
+        handler_names = subcommand.handler_map().keys()
+        if not any(name in cwords for name in handler_names):
+            for handler_name in subcommand.handler_map():
+                if handler_name.startswith(current):
+                    print(handler_name)
```

Comment:
```suggestion
        # Complete sub-actions (unless one is already given).
        if not any(name in cwords for name in subcommand.handler_map()):
            for handler_name in subcommand.handler_map()
                if handler_name.startswith(current):
                    print(handler_name)
```

Reply:
:+1: squashed that change in with the rest: [7a20ed1](https://github.com/pypa/pip/pull/13140/commits/7a20ed1f4314fe7b9331066ebc856c037dacdc71)